base/bucket_gocb.go

```diff
@@ -1721,7 +1735,7 @@ func (bucket *CouchbaseBucketGoCB) PutDDoc(docname string, value interface{}) er
 	case *sgbucket.DesignDoc:
 		sgDesignDoc = *typeValue
 	default:
-		return fmt.Errorf("CouchbaseBucketGoCB called with unexpected type.  Expected sgbucket.DesignDoc or *sgbucket.DesignDoc, got %T", value)
+		return RedactErrorf("CouchbaseBucketGoCB called with unexpected type.  Expected sgbucket.DesignDoc or *sgbucket.DesignDoc, got %T", MD(value))
```

Comment:
We don't want to convert`value` into Metadata, as we're capturing the type (`%T) here.